opentelemetry-instrumentation/src/opentelemetry/instrumentation/bootstrap_gen.py

```diff
@@ -156,6 +156,10 @@
         "library": "pymongo >= 3.1, < 5.0",
         "instrumentation": "opentelemetry-instrumentation-pymongo==0.51b0.dev",
     },
+    {
+        "library": "pymssql >= 2, < 3",
```

Comment:
I think these need to be regenerated with the new version restrictions.

Reply:
Done.